geometry_msgs/Point position
std_msgs/String type
---
